scenes/root/Root.gd

```diff
@@ -92,3 +108,22 @@ func _on_run_as_client_pressed():
 	var world = load("res://scenes/world/World.tscn").instantiate()
 	world.name = "World"
 	self.add_child(world)
+
+
+func _on_reset_pressed():
+	$SelectRunMode/VBoxContainer.hide()
+	$SelectRunMode/ComfirmResetContainer.show()
+
+
+func _on_reset_yes_pressed():
+	# Remove the database file if it exists to be in a clean state after this button was pressed
+	if FileAccess.file_exists(JSONDatabaseBackend.USERS_FILEPATH):
```

Comment:
When I was fixing this I bumped against the problem that when the reset is pressed the "J.server" instance does not exist yet.

What should we do with this function? Keep it as is, since it will only be used for debugging, or shall I just remove it to make the code clean?

Reply:
I removed the function. I don't think it adds any value to the game. If someone wants to do the reset, they can remove the persistent file.

Reply:
Hmmm, I didn't think about the lifecycle of the scenes. Yea I think its chill to just remove it for now.